react/features/video-layout/functions.any.ts

```diff
@@ -149,16 +149,6 @@ export function updateAutoPinnedParticipant(
     }
 }
 
-/**
- * Selector for whether we are currently in tile view.
- *
- * @param {Object} state - The redux state.
- * @returns {boolean}
- */
-export function isLayoutTileView(state: IReduxState) {
-    return getCurrentLayout(state) === LAYOUTS.TILE_VIEW;
```

Comment:
@saghul If the condition in isLayoutTileView applies, this is what actually happens inside getCurrentLayout
if (shouldDisplayTileView(state)) {
        return LAYOUTS.TILE_VIEW;